src/TokenCache.php

```diff
@@ -13,16 +14,25 @@ public function key(string $hashedToken): string
 
     public function get(string $hashedToken): ?AccessToken
     {
-        return Cache::get($this->key($hashedToken));
+        return $this->cache()->get($this->key($hashedToken));
     }
 
     public function put(AccessToken $token): bool
     {
-        return Cache::put($this->key($token->token), $token, now()->addDay());
+        return $this->cache()->put(
+            $this->key($token->token),
+            $token,
+            now()->addDay(),
+        );
     }
 
     public function forget($hashedToken): bool
     {
-        return Cache::forget($this->key($hashedToken));
+        return $this->cache()->forget($this->key($hashedToken));
+    }
+
+    private function cache(): Repository
+    {
+        return Cache::store(getenv('LARAVEL_OCTANE') ? 'octane' : null);
```

Comment:
This changes it from a cache that is shared between instances to one that is unique per instance/container. Not a problem, perhaps?

Reply:
On second thought...

The writing to the cache will take place on the api-instances. The storing of these values in the database will be scheduled and then executed by a worker? The worker will not have access to the octace cache, and even if it did (if we ran the worker on the same instance or used octane workers) it would just have access to a single instance of the cache, not the other instances?

Or may I completely misunderstanding how this works? :sweat_smile: